block/submit.go

```diff
@@ -40,22 +41,24 @@ func SubmitLoopInner(
 	ctx context.Context,
 	logger types.Logger,
 	bytesProduced chan int, // a channel of block and commit bytes produced
-	maxBatchSkew uint64, // max number of batches that submitter is allowed to have pending
+	maxBlockSkew uint64, // max number of blocks that submitter is allowed to have pending
+	pendingSubmittedBlocks func() uint64,
```

Comment:
not sure I get the name here.
I think you mean `pendingSubmissionBlocks`? 

cause `pending` and `submitted` are the contrary. 

also why not just `pendingBlocks` ? we're in the context of submission here anyway.